### PR TITLE
Fix/flag names

### PR DESCRIPTION
**Pull request by JohnRusk**
Reviewers: the changes, at least to test code, were more extensive that we would have liked, due to the changing of the block size to be specified in MiB.  

PR contains two commits: one with the block size changes, and one with all the other (simple) renames.

### CODE REVIEW

File: cmd/make.go [hunk 1]
```diff
@@ -214,6 +214,6 @@ func init() {
 		},
 	}
 
-	makeCmd.PersistentFlags().Uint32Var(&rawArgs.quota, "quota", 0, "specifies the maximum size of the share in gigabytes, 0 means you accept the file service's default quota.")
+	makeCmd.PersistentFlags().Uint32Var(&rawArgs.quota, "quota-gb", 0, "specifies the maximum size of the share in gigabytes, 0 means you accept the file service's default quota.")
```
Comment (zezha-msft) [inline cmd/make.go]: Clarify that it's GiB?

File: testSuite/scripts/test_upload_page_blob.py [hunk 2]
```diff
@@ -66,7 +66,7 @@ def test_page_blob_upload_partial_sparse_file(self):
         # execute azcopy pageblob upload.
         destination_sas = util.get_resource_sas(file_name)
         result = util.Command("copy").add_arguments(file_path).add_arguments(destination_sas).add_flags("log-level", "info"). \
-            add_flags("block-size", "4194304").add_flags("blob-type","PageBlob").execute_azcopy_copy_command()
```
Comment (zezha-msft) [inline testSuite/scripts/test_upload_page_blob.py]: wow that's a lot of manual changes.. we should definitely improve this later

File: cmd/make.go (hunk 1 above)
Comment (JohnRusk) [reply cmd/make.go]: Sound OK if I clarify only in the comments?  Alternative, naming it quota-gib seems odd to me somehow. (Not 100% sure why).

File: testSuite/scripts/test_upload_page_blob.py (hunk 2 above)
Comment (JohnRusk) [reply testSuite/scripts/test_upload_page_blob.py]: Agreed!  But not doing anything about it now ;-)

File: ChangeLog.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
 # Change Log
 
+## Version 10.0.10 (GA)
```
[the remark this answers is not in the document]
Comment (JohnRusk) [reply ChangeLog.md]: Done

File: cmd/make.go (hunk 1 above)
Comment (zezha-msft) [reply cmd/make.go]: it could be in the flag description, right?

File: testSuite/scripts/test_upload_page_blob.py (hunk 2 above)
Comment (zezha-msft) [reply testSuite/scripts/test_upload_page_blob.py]: :shipit:

File: cmd/make.go (hunk 1 above)
Comment (JohnRusk) [reply cmd/make.go]: Yes. I've put it in the description.  Which was what I meant, when I foolishly said "comments"

Comment (zezha-msft) [reply cmd/make.go]: Sounds good to me!

**Pull request merged**